packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts

```diff
@@ -180,7 +179,8 @@ export class FilamentViewer extends LitElement {
     // environment map with a single bright pixel that represents a 1 lux
     // directional light.
     if (lightingBaseName === 'spot1Lux') {
-      this[$directionalLight] = self.Filament.EntityManager.get().create();
+      const entity = self.Filament.EntityManager.get().create();
```

Comment:
An explicit variable name (e.g., `directionalLight`) would make this easier to read

Reply:
Done!